readmestaging20180813

```diff
@@ -0,0 +1 @@
+Creating branch staging20180813 for deploying to spokewfstagingdeploy
```

Comment:
cleanup

Reply:
removed -- it got in here because I merged WFP's staging into my branch